src/components/TeamLocations/TeamLocations.jsx

```diff
@@ -230,7 +233,7 @@ function TeamLocations() {
                   type="text"
                   className="form-control"
                   aria-label="Search"
-                  placeholder="Search Text"
+                  placeholder="Search by Location"
```

Comment:
**Suggestion**: It looks like the placeholder name is incorrect because the search functionality is currently set to search by location and username.

_Search by name:_
<img width="1499" alt="image" src="https://github.com/user-attachments/assets/0d0ed9d5-6d9f-4d75-80c2-3d80ae647d77">

_Search by location_
<img width="1499" alt="image" src="https://github.com/user-attachments/assets/c062197f-14b2-43a1-8b2e-55ee782f9a21">

Reply:
@kabraambika 
Thanks for Reviewing. Below are the resolved points.
1. Fixed the dark theme color for the search table. 
2. The Table view button error that occurs is not the scope of this PR. However you can raise this Bug issue in the web page, for developers to fix this.
3. Changed the text from "Search by Location" to "Search City/Country/Name"
Please pull the latest changes and review again.

Reply:
Thanks for the quick response and fix. I'll review the PR again.

Reply:
Below mentioned obervations are fixed.

- [ ] Dark mode is fixed as shown below:
<img width="1503" alt="image" src="https://github.com/user-attachments/assets/30c309cb-8348-45ae-9f63-0389c6a770c9">

- [ ] Placeholder text is fixed as well as shown below:
<img width="1504" alt="image" src="https://github.com/user-attachments/assets/a0b5e6b8-c179-466e-a3c1-1d6b0b12589a">